src/main.cpp

```diff
@@ -533,13 +533,6 @@ int main ( int argc, char** argv )
 
 
     // Dependencies ------------------------------------------------------------
-    // per definition: if we are in "GUI" server mode and no central server
```

Comment:
We need this here in case of a headless server since no settings are used in that case.

Reply:
The inifile is used if specified and settings are _always_ loaded.  If no GUI is given settings are no _written_ to the inifile.  So the load is always called.

Reply:
Rebase moved this to commit 2267a4d .